packages/@aws-cdk/aws-apigateway/README.md

```diff
@@ -455,6 +455,8 @@ books.addMethod('GET', new apigateway.HttpIntegration('http://amazon.com'), {
 });
 ```
 
+You can find a full working example [here](test/authorizers/integ.token-authorizer.ts).
```

Comment:
This relative link will work on GitHub, and be broken on the docs page.

Reply:
You are correct. I'll fix this up.

Weirdly enough, we have this everywhere - 

```shell
⇒  find . -name 'README.md' -not -path '*/node_modules/*' -exec grep -H '(test/' '{}' \;
./aws-dynamodb/README.md:[Example of configuring autoscaling](test/integ.autoscaling.lit.ts)
./aws-eks-legacy/README.md:[ssh into nodes example](test/example.ssh-into-nodes.lit.ts)
./aws-rds/README.md:[example of setting up a production oracle instance](test/integ.instance.lit.ts)
./aws-rds/README.md:[example of setting up master password rotation for a cluster](test/integ.cluster-rotation.lit.ts)
./aws-cloudfront/README.md:[create a distrubution with an default certificiate example](test/example.default-cert-alias.lit.ts)
./aws-cloudfront/README.md:[create a distrubution with an acm certificate example](test/example.acm-cert-alias.lit.ts)
./aws-cloudfront/README.md:[create a distrubution with an iam certificate example](test/example.iam-cert-alias.lit.ts)
./aws-s3/README.md:[sharing bucket between stacks](test/integ.bucket-sharing.lit.ts)
./aws-secretsmanager/README.md:[example of creating a secret](test/integ.secret.lit.ts)
./aws-ec2/README.md:[using NAT instances](test/integ.nat-instances.lit.ts)
./aws-ec2/README.md:[sharing VPCs between stacks](test/integ.share-vpcs.lit.ts)
./aws-ec2/README.md:[importing existing VPCs](test/integ.import-default-vpc.lit.ts)
./aws-ec2/README.md:[example of creating images](test/example.images.lit.ts)
./aws-ec2/README.md:[example of setting up VPC endpoints](test/integ.vpc-endpoint.lit.ts)
./aws-iam/README.md:[attaching permissions to role](test/example.role.lit.ts)
./aws-iam/README.md:[attaching policies to user and group](test/example.attaching.lit.ts)
./aws-iam/README.md:[attaching managed policies](test/example.managedpolicy.lit.ts)
./aws-iam/README.md:[supplying an external ID](test/example.external-id.lit.ts)
./aws-certificatemanager/README.md:[request a certificate example](test/example.simple-request.lit.ts)
./aws-certificatemanager/README.md:[request a validated certificate example](test/example.dns-validated-request.lit.ts)
./aws-codepipeline-actions/README.md:[example Pipeline to deploy CloudFormation](test/integ.cfn-template-from-repo.lit.ts)
./aws-codepipeline-actions/README.md:[Example of deploying a Lambda through CodePipeline](test/integ.lambda-deployed-through-codepipeline.lit.ts)
./aws-eks/README.md:[ssh into nodes example](test/example.ssh-into-nodes.lit.ts)
./aws-servicediscovery/README.md:[Creating a Cloud Map service within an HTTP namespace](test/integ.service-with-http-namespace.lit.ts)
./aws-servicediscovery/README.md:[Creating a Cloud Map service within a Private DNS namespace](test/integ.service-with-private-dns-namespace.lit.ts)
./aws-servicediscovery/README.md:[Creating a Cloud Map service within a Public namespace](test/integ.service-with-public-dns-namespace.lit.ts)
./aws-servicediscovery/README.md:[Creating a Cloud Map service within a Public namespace](test/integ.service-with-cname-record.lit.ts)
./aws-lambda/README.md:[Example of Lambda Code from Local Assets](test/integ.assets.lit.ts)
./aws-lambda/README.md:[Example of Lambda Layer usage](test/integ.layer-version.lit.ts)
./custom-resources/README.md:[construct](test/provider-framework/integration-test-fixtures/s3-file.ts) and
./custom-resources/README.md:[handler](test/provider-framework/integration-test-fixtures/s3-file-handler/index.ts).
./custom-resources/README.md:Checks that the textual contents of an S3 object matches a certain value. The check will be retried for 5 minutes as long as the object is not found or the value is different. See the source code for the [construct](test/provider-framework/integration-test-fixtures/s3-assert.ts) and [handler](test/provider-framework/integration-test-fixtures/s3-assert-handler/index.py).
./aws-logs/README.md:[retention example](test/example.retention.lit.ts)
./aws-logs/README.md:[metricfilter example](test/integ.metricfilter.lit.ts)
./aws-ssm/README.md:[using SSM parameter](test/integ.parameter-store-string.lit.ts)
./aws-ssm/README.md:[creating SSM parameters](test/integ.parameter.lit.ts)
./aws-kms/README.md:[sharing key between stacks](test/integ.key-sharing.lit.ts)
./aws-config/README.md:[example of setting up rules](test/integ.rule.lit.ts)
./aws-cloudformation/README.md:[example of Custom Resource](test/example.customresource.lit.ts)
./aws-autoscaling/README.md:[example of creating images](test/example.images.lit.ts)
```

Reply:
And it looks like, we're stripping it out during doc generation - https://docs.aws.amazon.com/cdk/api/latest/docs/aws-dynamodb-readme.html#configure-autoscaling-for-your-table vs https://github.com/aws/aws-cdk/blob/master/packages/%40aws-cdk/aws-dynamodb/README.md#configure-autoscaling-for-your-table.

Is this intentional?

Reply:
Hmm, unclear.

The right thing to do would probably be to change the relative links to absolute links during the doc site generation.